gradle/dependencies.gradle

```diff
@@ -27,8 +27,9 @@ ext {
       autoValue                 : '1.5.4',
       autoValueParcel           : '0.2.5',
       junit                     : '4.12',
-      supportLibVersion         : '1.0.0',
+      supportLibVersion         : '1.1.0',
```

Comment:
this version is required by SnackBar so that we could call `snackbar.setAnchorView` in NavigationView.